logging-api/src/main/java/com/cisco/oss/foundation/logging/transactions/SchedulerLogger.java

```diff
@@ -47,6 +56,20 @@ public static void success(String schedulerResult) {
 
     }
 
+    public static void successAsync(SchedulerLogger schedulerLogger) {
+
+        FlowContextFactory.deserializeNativeFlowContext(TransactionLogger.getFlowContextAsync(schedulerLogger));
```

Comment:
I think that would be an improvment, but doesn't look possible because the function is static, so it can't reference any internal fields.
Notice I added a non-static successAsync which does what you suggested.

One little improvment i could make would be to get the flow context with the non-static getter I've added as part of the commit:
    schedulerLogger.getFlowContextAsync()
instead of
    TransactionLogger.getFlowContextAsync(schedulerLogger)